src/app/shared/table/actions/table-export-as-csv-action.ts

```diff
@@ -0,0 +1,43 @@
+import { TranslateService } from '@ngx-translate/core';
+import { DialogService } from 'app/services/dialog.service';
+import { ChargingStation, OcppParameter } from 'app/types/ChargingStation';
+import { ButtonAction } from 'app/types/GlobalType';
+import { ButtonColor, ButtonType, TableActionDef } from 'app/types/Table';
+import { Constants } from 'app/utils/Constants';
+import { Utils } from 'app/utils/Utils';
+import * as FileSAver from 'file-saver';
+
+import { TableAction } from './table-action';
+
+export class TableExportAsCSVAction implements TableAction {
```

Comment:
Same as above TableExportOCPPAsCSVAction
This class should be in dir 'table-actions' in charging station dir instead of shared.
Extends the TableExportAction class (even if there is no shared code yet, at least you will share the icons, label, tooltip...)